brave/brave-core
# Make NTP setting menu keyboard accessible

fixes https://github.com/brave/brave-browser/issues/5041
fixes https://github.com/brave/brave-browser/issues/5199
~Brave-ui PR: https://github.com/brave/brave-ui/pull/506~
Quick View:
![keyboard-settings](https://user-images.githubusercontent.com/9125231/60218170-622e7580-9823-11e9-8259-6931c9d7774e.gif)


## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan 1:
- Go to new tabs page 
- Press tab 
- There should be an outline when you reach the settings icon at the bottom left of the page
- Pressing `space` or `enter` will open the setting menu
- Pressing tab will allow you to select the settings (i.e. show background image)
- Pressing `space` or `enter` or 'escape'  will allow you to close the menu
- Clicking the settings icon while menu is open will also close the menu

## Test Plan 2:
- Open new tabs page
- Open customization menu at the bottom right corner 
- Open another NTP and assert customization menu is closed
- Close browser with customization menu open; NTP on a fresh browser instance should be closed

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
